src/API/Plugins.php

```diff
@@ -431,6 +445,48 @@ public function activate_plugins( $request ) {
 		) );
 	}
 
+	/**
+	 * Return recommended payment plugins.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return \WP_Error|\WP_HTTP_Response|\WP_REST_Response
+	 */
+	public function recommended_payment_plugin( $request ) {
+		// Default to marketing category (if no category set).
+		$all_plugins        = PaymentPlugins::get_instance()->get_recommended_plugins();
+		$valid_plugins      = [];
+		$per_page           = $request->get_param( 'per_page' );
+		$locale             = get_locale();
+		$suggestion_locales = array(
+			'en_AU',
+			'en_CA',
+			'en_GB',
+			'en_NZ',
+			'en_US',
+			'en_ZA',
+		);
```

Comment:
Not saying we need to utilize this, but I was searching for an existing function that might hold this business logic: [`WC_Marketplace_Suggestions::allow_suggestions()`](https://github.com/woocommerce/woocommerce/blob/master/includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php#L161)

Reply:
Yeah that's where I took it from. My logic is currently a little different, to allow for new languages more easily, but I suppose I could leave that out.